src/main/resources/templates/index.html

```diff
@@ -19,7 +18,7 @@
 
 <body>
     <div class="container-fluid col-md-6">
-        <img class="img-fluid mx-auto d-block h-25" src="../static/img/civic-type-r.jpg" alt="Honda Civic Type-R">
```

Comment:
the content of the static file are available automatically in Spring Boot:
https://spring.io/blog/2013/12/19/serving-static-web-content-with-spring-boot

Reply:
Good catch.

Reply:
This was later implemented soon after the review.